arel/visitors/to_sql.rb

```diff
@@ -0,0 +1,3 @@
+def compile(node, &block)
```

Comment:
@VanjaJeftic what does this do and where it's used?

Reply:
@basti I'm not completely sure what exactly it is for, I searched the internet a lot because I had an error related to that arel folder and it's clear to me that it has something to do with the database, but I'm not sure exactly what kind. The error disappeared when I put the arel folder in the root directory. That folder was generated at some point when I was working with active storage.

Reply:
@VanjaJeftic can you try to remove this file and let me know what error happens? It's highly unusual to add a file like this. I don't think I ever had a `arel` dir in Rails root.